pkg/operator/etcdcertsigner/etcdcertsignercontroller.go

```diff
@@ -99,10 +102,23 @@ func (c *EtcdCertSignerController) syncAllMasters(recorder events.Recorder) erro
 
 	errs := []error{}
 	for _, node := range nodes {
-		if err := c.createSecretForNode(node, recorder); err != nil {
+		nodeInternalIPs, err := dnshelpers.GetInternalIPAddressesForNodeName(node)
+		if err != nil {
 			errs = append(errs, err)
+			continue
+		}
+		if err := c.ensureAndValidateSecrets(node, nodeInternalIPs, recorder); err != nil {
+			if errors.IsNotFound(err) {
```

Comment:
(No action required) When evaluating more than nil, maybe don't use the inline conditional check?

```golang
err := ...
if errors.IsNotFound(err) {
 ...
} else if err != nil {
 ...
}

```

This message brought to you by the society for the minimization of indentation.